apis/swagger.yml

```diff
@@ -337,6 +337,41 @@ paths:
           schema:
            $ref: "#/definitions/Error"
       tags: ["Container"]
+
+  /containers/{id}/stop:
+    post:
+      summary: "Stop a container"
+      operationId: "ContainerStop"
+      parameters:
+        - name: "id"
+          in: "path"
+          required: true
+          description: "ID or name of the container"
+          type: "string"
+        - name: "t"
+          in: "query"
+          description: "Number of seconds to wait before killing the container"
+          type: "integer"
+      responses:
+        204:
+          description: "no error"
+        304:
+          description: "container already stopped"
+          schema:
+            $ref: "#/definitions/Error"
```

Comment:
I know that this is from Moby's swagger.yml. While I am afraid status code 3xx is not an error. 
We can have a discussion that whether it is proper to set 304 here? 

We can take "pause an paused container" as an example. WDYT?

Reply:
OK! I will update